content/en/tracing/tracing_without_limits/_index.md

```diff
@@ -0,0 +1,220 @@
+- One-click search queries on any span or tag directly from the trace panel view.
+- Add, remove, and sort columns from span tags for a customized view.
+
+{{< img src="tracing/live_search/livesearchmain.gif" alt="Live Search" >}}
+
+The number of received spans per second is displayed at the top of the traces table, as well as the sampling rate. Since a stream of thousands of spans per second is not human readable, high throughput span streams are sampled for visual clarity but are still searchable. Use the Live Search query bar filtering features to filter the spans stream and the **Pause/Play** button at the top right of the screen to pause or resume the stream.
+
+`Live Search` mode is the default experience on the Traces page or you can choose the `LIVE` option in the time range selector to switch to the Live Search mode for the past 15 minutes from the `HISTORICAL` mode.
+
+**Note**: Selecting any span pauses the stream and displays more details about the selected span in the trace side panel.
+
+### Historical Search Mode
+
+{{< img src="tracing/live_search_and_analytics/tracing_without_limits_lifecycle-4.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="Trace Journey" >}}
+
+Historical Search Mode is accessed in the same way as Live Search mode.  To switch between the two, change the time selector to any period of time greater than 15 minutes.  This will change the data being searched from live search mode to indexed spans.
+
+In Historical Search Mode, the data searched is Indexed Spans, along with the root span of any trace with at least one indexed span.  These spans are kept by Datadog for 15 days from being indexed.
+
+{{< img src="tracing/live_search/livesearch_mode.gif" alt="Live Search mode" >}}
+
+**Note:** The entire associated trace will appear whenever you are viewing the flame graph associated with any indexed span, but only Indexed spans power searches in historical search mode.
+
+For example, if you filter by a tag that only appears on un-indexed spans, your search will return no results.
+
+What spans are indexed and at what retention rates are fully customizable. By default, [Datadog Intelligent Sampling](#datadog-intelligent-sampling-filter) will be applied to decide what spans to index, and this is the data powering Historical queries. To learn more about the default span retention filter and how to create your own additional filters, visit the [Indexing](#indexing-controls) section of this page, or the [Span Indexing][2] page within Datadog.
+
+### Filtering the Trace Stream and Search Query
+{{< img src="tracing/live_search/toplevespan.gif" alt="Live Search query" >}}
+
+A valid query in the search bar displays traces that match your search criteria across **all spans**. The search syntax is the same in the Live Search views as in the other trace views, but here, your query is matched against all of the ingested traces across any span and any tag, and not just the indexed ones.
```

Comment:
We've switched back to talking about Live Stream here, right? We should start this sentence with something like "In Live Search mode..." because we were previously talking about Historical.

Reply:
Worth just moving this section above Historical Search?